applications/ray/variables.tf

```diff
@@ -42,6 +42,11 @@ variable "ray_namespace" {
   default     = "myray"
 }
 
+variable "enable_ray_dashboard" {
```

Comment:
can we call it enable_grafana_on_ray_dashboard? it's not used anywhere else, right?